src/routes/safe/components/Transactions/helpers/TxParametersDetail/index.tsx

```diff
@@ -99,43 +128,142 @@ export const TxParametersDetail = ({
           </StyledText>
 
           <TxParameterWrapper>
-            <ColoredText
-              size="lg"
-              isOutOfOrder={isTxNonceOutOfOrder}
-              color={areSafeParamsEnabled(parametersStatus || defaultParameterStatus) ? 'text' : 'secondaryLight'}
-            >
+            <ColoredText size="lg" isOutOfOrder={isTxNonceOutOfOrder} color={color}>
               Safe nonce
             </ColoredText>
-            <ColoredText
-              size="lg"
-              isOutOfOrder={isTxNonceOutOfOrder}
-              color={areSafeParamsEnabled(parametersStatus || defaultParameterStatus) ? 'text' : 'secondaryLight'}
-            >
+            <ColoredText size="lg" isOutOfOrder={isTxNonceOutOfOrder} color={color}>
               {txParameters.safeNonce}
             </ColoredText>
           </TxParameterWrapper>
 
           {showSafeTxGas && (
             <TxParameterWrapper>
-              <Text
-                size="lg"
-                color={areSafeParamsEnabled(parametersStatus || defaultParameterStatus) ? 'text' : 'secondaryLight'}
-              >
+              <Text size="lg" color={color}>
                 SafeTxGas
               </Text>
-              <Text
-                size="lg"
-                color={areSafeParamsEnabled(parametersStatus || defaultParameterStatus) ? 'text' : 'secondaryLight'}
-              >
+              <Text size="lg" color={color}>
                 {txParameters.safeTxGas}
               </Text>
             </TxParameterWrapper>
           )}
           <StyledButtonLink color="primary" textSize="xl" onClick={onEdit}>
             Edit
           </StyledButtonLink>
+          {storedTx?.length > 0 && <TxAdvancedParametersDetail tx={storedTx[0]} />}
         </AccordionDetailsWrapper>
       </AccordionDetails>
     </Accordion>
   )
 }
+
+const TxParameterEndWrapper = styled.span`
+  display: flex;
+  justify-content: flex-end;
+  gap: 4px; // EthHashInfo uses a gap between the address and copy button
+`
+
+const TxAdvancedParametersDetail = ({ tx }: { tx: Transaction }) => {
+  const { txData, detailedExecutionInfo } = tx?.txDetails || {}
+
+  if (!txData || !detailedExecutionInfo) {
+    return null
+  }
+
+  const { value, to, operation, hexData } = txData
+  const { safeTxHash, baseGas, gasPrice, gasToken, refundReceiver, confirmations } =
+    (isMultiSigExecutionDetails(detailedExecutionInfo) && detailedExecutionInfo) || {}
+
+  return (
+    <>
+      <StyledDivider />
+
+      {value && (
+        <TxParameterWrapper>
+          <Text size="lg">value</Text>
+          <Text size="lg">{value}</Text>
+        </TxParameterWrapper>
+      )}
```

Comment:
These six lines repeat for every parameter.

I would make a local component `TxParam` that would take either a string (for simple text values), or an Element (for e.g. EthHashInfo).